docker-compose.fixture.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  puppet:
+    environment:
+      - CERT_VOLUME=puppetserver-config
```

Comment:
This is a little different than PE ... need to make sure the `CERT_DESTDIR` makes sense